src/multi_agent/tools/adapter.py

```
@@ -229,6 +228,44 @@ def validate_tools(self, tool_names: Set[str]) -> bool:
+            # Anthropic expects: {name, description, input_schema: {type, properties, required}}
+            anthropic_schema = {
+                "name": tool.name,
+                "description": tool.description,
+                "input_schema": {
+                    "type": "object",
+                    "properties": pydantic_schema.get("properties", {}),
+                    "required": pydantic_schema.get("required", [])
+                }
+            }
+
+            return anthropic_schema
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)